game/script-jokes.rpy

```diff
@@ -986,3 +1011,73 @@ label joke_shoemakers:
     extend 1csrsl " {i}tanned{/i}."
 
     return
+
+label joke_lead_times:
+    n 1ccsfl "...Alright.{w=0.75}{nw}"
+    extend 1csqem " You asked for this,{w=0.2} [player]."
+    n 1csqsl "Why is it so hard to land a job as a dog walker nowadays?"
+    n 1cslsl "..."
+    n 1ccssl "Heh."
+    n 1cdlfl "...Because there's such a long{w=0.5}{nw}" 
+    extend 1fslfl " {i}lead time{/i}."
+
+    return
+
+label joke_shark_literature:
+    n 1fcsbs "Alright!{w=0.75}{nw}"
+    extend 1fdwbg " How about this one,{w=0.5}{nw}" 
+    extend 1fsqsm " [player]?"
+    n 1fcsbg "What kind of literature do you hand a shark?"
+    n 1fnmsm "..."
+    n 1usqss "No?{w=0.75}{nw}"
+    extend 1fsqsm " Not even a guess?"
+    n 1fcssmesm "How disappointing.{w=0.75}{nw}"
+    extend 1fcsbg " Isn't it obvious,{w=0.2} [player]?"
+    n 1fnmss "You give them stuff they can really...{w=1}{nw}" 
+    extend 1fsqbg " {i}sink their teeth into{/i},{w=0.75}{nw}" 
+    extend 1fchgn " duh!"
+    
+    return
+
+label joke_developers_committed:
+    n 1ttrpu "You know,{w=0.2} [player]...{w=1}{nw}"
+    extend 1tlraj " I've always wondered."
+    n 1tsqfl "Is it hard for developers to start relationships?{w=0.75}{nw}"
+    extend 1tllbo " The ones that mess around with code and all that stuff?"
+    n 1tsqsl "..."
+    n 1tsqfl "No?{w=0.75}{nw}"
+    extend 1clrpu " Huh.{w=1}{nw}"
+    extend 1csqss " You sure,{w=0.2} [player]?"
+    n 1fcsss "'Cause from what I'm reading here..."
+    n 1nchgn "...They always seem to be pretty {i}committed{/i} already!"
+
+    return
+
+label joke_shelved_plans:
+    n 1ccstr "I gotta say,{w=0.2} [player].{w=0.75}{nw}"
+    extend 1cslca " I'm still pretty bummed out about being stuck here and everything,{w=0.2} you know."
+    n 1cllbo "..."
+    n 1tnmfl "What?{w=0.75}{nw}"
+    extend 1tnmbo " Didn't I tell you?"
+    n 1fcsgs "I actually landed an interview for a part-time job after school at a bookstore!{w=0.75}{nw}"
```

Comment:
Tbh I was a bit worried first, that the tone is a bit too serious. But on the other hand I think it is interesting to see if players discuss if Natsuki is being honest here or just telling a story to fit to the joke.

Reply:
I like wrapping little stories into jokes Nat actually enjoys/is neutral to! She's a little cocky and likes to show off a bit, so it feels right that she'd use some storytelling/showmanship to really sell the joke and catch someone off guard!

I would also say that given she's literally reading the jokes from a book, that alone should be enough for the average player to reason she's telling a joke rather than an actual story